app/upload/rest.py

```diff
@@ -45,6 +45,7 @@ def get_paginated_uploads(service_id, limit_days, page):
                 "%Y-%m-%d %H:%M:%S") if upload.scheduled_for else upload.created_at.strftime("%Y-%m-%d %H:%M:%S"),
             'upload_type': upload.upload_type,
             'template_type': None,
```

Comment:
template_type can just be upload.template_type, then you can remove line 63 where it's set for jobs only.

Reply:
Changed in f57b544